internal/testing/testutils/stdio.go

```diff
@@ -0,0 +1,33 @@
+// Copyright © 2025 Ping Identity Corporation
+
+package testutils
+
+import (
+	"io"
+	"os"
+)
+
+// CaptureStdout executes a function and returns its standard output as a string.
+func CaptureStdout(f func()) string {
```

Comment:
I realize this is test code so maybe not important, but any risk of this blocking forever? Would it be worth requiring `f` to accept a cancelation context?